customer-commons/src/test/java/no/unit/nva/customer/model/CustomerDtoTest.java

```diff
@@ -62,4 +63,8 @@ private CustomerDto randomCustomer() {
     private Collection<VocabularyDto> randomVocabularies() {
         return List.of(randomVocabulary(), randomVocabulary(), randomVocabulary());
     }
+
+    private ApplicationDomain randomApplicationDomain() {
+        return ApplicationDomain.NVA;
```

Comment:
This introduces a bug, since it isn't random…when another is added, use the randomElement(ArrayList.whateverTheMethodIs(ApplicationDomain.values)) trick.

Reply:
fixed